ignite/internal/tools/gen-mig-diffs/main.go

```diff
@@ -0,0 +1,15 @@
+package main
```

Comment:
Given that this tool is standalone, shall we not extract it in its own go.mod?
This will avoid adding extra dependencies on ignite.
Especially because two of the added dependencies/packages: https://github.com/hexops/gotextdiff and https://github.com/gobwas/glob aren't maintained anymore.